microsoft/vscode-python-devicesimulator
# Removing dependency on pythonPath setting

# Description:

The Python extension is deprecating the python.pythonPath setting, which the DSX extension relies on. 

## Type of change

Please delete options that are not relevant.


# Limitations:

- No tests 
- Limited support for multi-root workspaces (interpreter path is retrieved for the first workspace folder, instead of the workspace folder from the file in focus) 

# Testing:

This is what I've done to test the changes:

- Run "Preferences: Open Settings (JSON)" command and add the following line:
```json
 "python.experiments.optInto": ["DeprecatePythonPath - experiment"]
```

- Run "Developer: Reload Window"

- Added a breakpoint to line 183 of src\service\setupService.ts to see the value of originalPythonExecutablePath changing to the right environment that was selected in the workspace. 

# Checklist:

- [x] My code has been formatted with `npm run format` and passes the checks in `npm run check`
